dotcom-rendering/src/server/prod-server.ts

```diff
@@ -41,6 +48,24 @@ export const prodServer = (): void => {
 	app.use(express.json({ limit: '50mb' }));
 	app.use(compression());
 
+	app.use((req, res, next) => {
+		res.on('finish', () => {
+			const pageId = getPageIdFromBody(req.body);
+			logger.info(`${req.method} ${req.path} ${pageId}`, {
+				pageId,
+				req: {
+					path: req.path,
```

Comment:
As per above, I know this replicates the naming in express, but we do use `req.url` elsewhere, proliferating logging names can be confusing when searching in kibana.